libeos-updater-util/avahi-service-file.h

```diff
@@ -44,4 +44,241 @@ gboolean eos_avahi_service_file_delete (const gchar   *avahi_service_directory,
                                         GCancellable  *cancellable,
                                         GError       **error);
 
+/* new DNS-SD records format for ostree */
+/* TODO: gtkdocize it? */
+/* TXT records' values are basically serialized GVariants. Below all
+ * the keys and their GVariant types are described.
+ *
+ * The TXT records are served for the service type
+ * "_repo._ostree._tcp".
+ *
+ * Common fields for all versions of TXT records:
+ *
+ * - version, describes the version of the TXT records format:
+ *   - key: "v"
+ *   - type: "y" (byte)
+ *   - contents: a version number (note: it is not an ascii digit)
+ *
+ * Fields for version 1 of TXT records:
+ *
+ * - refs bloom filter, a Bloom filter that contains all the refs the host has
+ *   - key: "rb"
+ *   - type: "(yyay)" (tuple containing a byte, a byte and an array of bytes)
+ *   - contents: first byte is the "k" parameter, second byte is the
+ *               hash id, an array of bytes are the bloom filter bits
+ *
+ * - repository index, a number identifying an OSTree repository
+ *   - key: "ri"
+ *   - type: "q" (uint16)
+ *   - contents: it gets appended to the host's URL as "/%u"; 0 is a special
```

Comment:
Alright, it's big endian from now on. I updated the two places where we create gvariants for repo index and summary timestamp. Will file a PR for your ostree branch later with the changes.